src/d2q9/Dynamics.R

```diff
@@ -32,10 +32,21 @@ AddSetting(name="omega", comment='one over relaxation time')
 AddSetting(name="nu", omega='1.0/(3*nu + 0.5)', default=1.6666666, comment='viscosity')
 AddSetting(name="Velocity", default=0, comment='inlet/outlet/init velocity', zonal=T)
 AddSetting(name="Density", default=1, comment='inlet/outlet/init density', zonal=T)
-
+AddSetting(name="GravitationY", comment='Gravitation in the direction of y')
+AddSetting(name="GravitationX", comment='Gravitation in the direction of x')
 # Globals - table of global integrals that can be monitored and optimized
 
 AddGlobal(name="PressureLoss", comment='pressure loss', unit="1mPa")
 AddGlobal(name="OutletFlux", comment='pressure loss', unit="1m2/s")
 AddGlobal(name="InletFlux", comment='pressure loss', unit="1m2/s")
 
+
+AddSetting(name="S0", default="0", comment='MRT Sx')
+AddSetting(name="S1", default="0",comment='MRT Sx')
+AddSetting(name="S2", default="0",comment='MRT Sx')
+AddSetting(name="S3", default="-.333333333", comment='MRT Sx')
+AddSetting(name="S4", default="0", comment='MRT Sx')
+AddSetting(name="S5", default="0", comment='MRT Sx')
+AddSetting(name="S6", default="0", comment='MRT Sx')
+AddSetting(name="S7", default="1.-omega", comment='MRT Sx')
```

Comment:
This cannot work. default value have to be an absolute value - not related to other settings.